testsuite/MDAnalysisTests/analysis/test_hbonds.py

```diff
@@ -28,9 +28,10 @@
 import pytest
 from MDAnalysis import SelectionError
 
-from numpy.testing import assert_equal, assert_array_equal, assert_almost_equal, assert_array_almost_equal
+from numpy.testing import assert_equal, assert_array_equal, assert_almost_equal, assert_array_almost_equal,assert_allclose
```

Comment:
```
from numpy.testing import (assert_equal, assert_array_equal, assert_almost_equal, 
                           assert_array_almost_equal,assert_allclose)
```
or the thought police (a.k.a flake8) will complain

Reply:
This refers to [PEP8](https://www.python.org/dev/peps/pep-0008/) a common style guide for python. We try hard to follow it but do not enforce it. But please make the suggested change.